x-pack/plugins/case/server/services/index.ts

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  KibanaRequest,
+  KibanaResponseFactory,
+  Logger,
+  SavedObject,
+  SavedObjectsClientContract,
+  SavedObjectsFindResponse,
+  SavedObjectsUpdateResponse,
+  SavedObjectReference,
+} from 'kibana/server';
+import { CASE_COMMENT_SAVED_OBJECT, CASE_SAVED_OBJECT } from '../constants';
+import {
+  NewCaseFormatted,
+  NewCommentFormatted,
+  UpdatedCaseType,
+  UpdatedCommentType,
+} from '../routes/api/types';
+import { PluginSetupContract as SecurityPluginSetup } from '../../../security/server';
+import { AuthenticatedUser } from '../../../security/common/model';
```

Comment:
@azasypkin should we export this from `plugins/security/server` and `plugins/security/public` if we're treating this as part of our public contract? Or is importing from `plugins/security/common/model` here acceptable?

Reply:
> @azasypkin should we export this from plugins/security/server and plugins/security/public if we're treating this as part of our public contract?

Yep, definitely! That's the only place other plugins are allowed to import from.

Reply:
@stephmilovic would you be willing to make this change as part of this PR? It'll likely be faster than waiting on us to get a PR in ahead of this.